src/main/java/blackjack/domain/participant/Players.java

```diff
@@ -0,0 +1,41 @@
+package blackjack.domain.participant;
+
+import blackjack.domain.card.HandGenerator;
+import blackjack.exception.InvalidPlayerCountException;
+import java.util.List;
+
+public class Players {
+    private final List<Player> players;
+    private int order = 0;
+
+    public Players(List<Name> playerNames, HandGenerator handGenerator) {
+        checkPlayersEmpty(playerNames);
+        this.players = playerNames.stream()
+                .map(playerName -> new Player(playerName, handGenerator))
+                .toList();
+    }
+
+    private void checkPlayersEmpty(List<Name> playerNames) {
+        if (playerNames.isEmpty()) {
+            throw new InvalidPlayerCountException();
+        }
+    }
+
+    public boolean hasNext() {
+        return order < players.size();
+    }
+
+    public void increaseOrder(PlayerAction playerAction) {
+        if (playerAction.equals(PlayerAction.STAND) || !getPlayerAtOrder().canHit()) {
```

Comment:
```suggestion
        if (playerAction.isStand() || !getPlayerAtOrder().canHit()) {
```
또한 enum 비교는 `==`로도 가능합니다~!